### PR TITLE
docs(material/chips): update a11y documention on checkmark indicators

### PR DESCRIPTION
**Pull request by zarend**
Update the accessibility section on checkmark indicators for single-selection. Add instructions to always communicate selection with icon indicators. Fulfill documentation needs as follow-up for #25890 and
 #26338.

### CODE REVIEW

File: src/material/chips/chips.md [hunk 1]
```diff
@@ -187,3 +187,7 @@ For both MatChipGrid and MatChipListbox, always apply an accessible label to the
 Always apply MatChipRemove to a `<button>` element, never a `<mat-icon>` element.
 
 When using MatChipListbox, never nest other interactive controls inside of the `<mat-chip-option>` element. Nesting controls degrades the experience for assistive technology users.
+
+`MatChipListbox` visually communicates selected state with both change of color and icon indicators
+By default, single-selection displays checkmark on selected option. Always shows icon indicators
```
Comment (crisbeto) [inline src/material/chips/chips.md:192]: The punctuation seems incorrect between "option" and "Always".

**Comment by zarend** [comment]
@crisbeto I reworded this. Do the punctuation and wording look better than before?

File: src/material/chips/chips.md [hunk 2]
```diff
@@ -187,3 +187,5 @@ For both MatChipGrid and MatChipListbox, always apply an accessible label to the
 Always apply MatChipRemove to a `<button>` element, never a `<mat-icon>` element.
 
 When using MatChipListbox, never nest other interactive controls inside of the `<mat-chip-option>` element. Nesting controls degrades the experience for assistive technology users.
+
+By default, `MatChipListbox` shows a selection indicator, even for single-selection. You can hide the indicator using `hideSingleSelectionIndicator`, but showing the indicator ensurers selection is clearly visible to all users.
```
Comment (mmalerba) [inline src/material/chips/chips.md:191]: Maybe something like this?
```md
In order to guarantee and accessible experience, `MatChipListbox` shows a selection indicator in both 
multi-selection and single-selection mode. You can disable the indicator for single-selection mode using
`hideSingleSelectionIndicator`. If you disable the indicator, you may want to add some additional signal of
the selected state to the user (see [whatever aria page talks about this](...))
```

File: src/material/chips/chips.md [hunk 3]
```diff
@@ -187,3 +187,5 @@ For both MatChipGrid and MatChipListbox, always apply an accessible label to the
 Always apply MatChipRemove to a `<button>` element, never a `<mat-icon>` element.
 
 When using MatChipListbox, never nest other interactive controls inside of the `<mat-chip-option>` element. Nesting controls degrades the experience for assistive technology users.
+
+By default, `MatChipListbox` displays a checkmark to identify selected items. While you can the checkmark indicator for single-selection via `hideSingleSelectionIndicator`, this makes your component less accessible by making it harder or impossible for users to visually identify selected items.
```
Comment (mmalerba) [inline src/material/chips/chips.md:191]: "can the checkmark" => "can disable the checkmark"

"this makes your component" => "this makes the component"

**Review by mmalerba — APPROVED** [review]
LGTM

File: src/material/chips/chips.md (hunk 3 above)
Comment (zarend) [reply src/material/chips/chips.md]: fixed the typos

**Pull request merged**